.github/workflows/github-actions-push-image.yml

```diff
@@ -2,13 +2,11 @@ name: Publish Docker image
 
 on:
   push:
-    paths:
-      - '**'
-      - '!README.md'
-      - '!.github/**/*.md'
-      - '!.github/workflows/**'
-      - '!.env.example'
-
```

Comment:
Hi @yurnov 
I added these ignores to stop publishing the docker image if the code wasn't changed.
Please explain why it was removed

Reply:
> Hi @yurnov I added these ignores to stop publishing the docker image if the code wasn't changed. Please explain why it was removed

It's not a real PR, it's just an attempt to trigger CI to ensure that no security vulnerabilities here. Just ignore